### PR TITLE
ACN_CD_CrowdStrikeFalcon_Connector01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

+ CrowdStrike Falcon Endpoint Protection Data Connector

### CODE REVIEW

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "CrowdStrikeFalconEventStream",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\""
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Hosts with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstHostName /n | top 10 by count_"
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:14]: Please replace /n with \n in query for next line.

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    }, {
+        "description": "Top 10 Users with Detections",
+        "query": "CrowdStrikeFalconEventStream \n | where EventType == \"DetectionSummaryEvent\" \n| summarize count() by DstUserName /n | top 10 by count_"
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:17]: Please replace /n with \n in query for next line.

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+    }],
+    "dataTypes": [{
+        "name": "CommonSecurityLog (CrowdStrikeFalconEventStream)",
+        "lastDataReceivedQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\"\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+    }],
+    "availability": {
+        "status": 1
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:28]: Can you please add "isPreview": true, please refer the template here https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 1 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: corrections made

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 2 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: corrections made

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 3 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: added

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 4]
```diff
@@ -0,0 +1,95 @@
+{
+    "id": "CrowdStrikeFalconEndpointProtection",
+    "title": "CrowdStrike Falcon Endpoint Protection",
+    "publisher": "CrowdStrike",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "CrowdStrikeFalconEventStream",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\""
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:10]: There is a special character next to "CommonSecurityLog" (in 3 places), Can you please remove that. Please check the attached screenshot that gives the placing.
CrowdStrike-SpecialCharacters

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 5]
```diff
@@ -0,0 +1,95 @@
+{
+    "id": "CrowdStrikeFalconEndpointProtection",
+    "title": "CrowdStrike Falcon Endpoint Protection",
+    "publisher": "CrowdStrike",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:6]: Please use https://aka.ms/sentinel-crowdstrikefalconendpointprotection-parser for parser redirection link.

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+            "scope": "Workspace",
+            "requiredPermissions": {
+                "action": true
+            }
+        }]
+    },
+    "instructionSteps": [
+        { 
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow the steps](https://aka.ms/sentinel-crowdstrikefalcon-parser) to use the Kusto function alias, **CrowdStrikeFalconEventStream**",
```
Comment (v-jayakal) [inline DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:55]: Please use https://aka.ms/sentinel-crowdstrikefalconendpointprotection-parser for parser redirection link.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 6 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: resolved

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 5 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: resolved

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 4 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json:10]: @v-jayakal  I wasnt able to see if same. I deleted the space and added it, let me know if that fixed it

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json (hunk 4 above)
Comment (chicduong) [reply DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json]: updated made

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**